interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props is short for properties in react and is the part containing states and elements such as arrays to be passed to other portions of the code.
 
-  Researched answer:
+  Researched answer: Numbers, booleans, arrays, variables, etc can be put into props as a form of global elements that can be used in components and code blocks in a react app.
```

Comment:
Props are helpful in passing that information from the parent to the child component.